src/Header.php

```diff
@@ -0,0 +1,738 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Http;
+
+/**
+ * HTTP headers
+ * @link https://developer.mozilla.org/docs/Web/HTTP/Headers
+ */
+final class Header
+{
+    /**
+     * Authentication
```

Comment:
Docblock should be for something but this one is not attached to any element. Same for other sections.

Either it should be just block comment such as `/* */` or inline comment such as `//` or should be removed.